demo/src/app/components/accordion/overview/accordion-overview.component.html

```diff
@@ -0,0 +1,114 @@
+<p>
+  Accordion is a component that allows you to organize sections of the user interface into collapsable panels.
+</p>
+
+
+<ngbd-overview-section [section]="sections['basic-usage']">
+  <p>
+    In the most primitive form, an accordion is used by creating an <code>ngb-accordion</code> element, with nested <code>ngb-panel</code> elements within it.
+  </p>
+  <p>
+    This example might look something like:
+    <ngbd-code [snippet]="snippets.basicNoOpts"></ngbd-code>
+  </p>
+
+  <p>
+    The above example uses the <code>title</code> attribute for the ngb-panel element to define the text displayed on the panel
+    header. While this can be customized, this setup provides the basic representation of the title as clickable text.
+  </p>
+
+  <p>
+    The above example also uses the <code>ngbPanelContent</code> directive to denote the container of the content to be displayed in the panel.
+  </p>
+
+  <p>
+    Some additional notes around basic usage of the component are the following:
+  </p>
+
+  <ul>
+    <li>
+      The accordion element can be given a template reference variable, which allows external controls to drive the
+      operation of the accordion (i.e., which panels are open).
+    </li>
+    <li>
+      Each <code>ngb-panel</code> can be given an <code>id</code> attribute, which would allow for direct access to
+      identify or toggle the open status of the given panel.
+    </li>
+    <li>
+      It is also possible to assign the <code>activeIds</code> attribute of the <code>ngb-accordion</code> element to the
+      value of one of the specified <code>ngb-panel</code> element's <code>id</code> to define which panel is open by default.
+    </li>
+    <li>
+      A panel can also be disabled, which prevents it from being able to be opened by using the <code>disabled</code> attribute.
+    </li>
+
+    <li>
+      The accordion itself can also be set to only allow one open panel at a time by using the <code>closeOthers</code> attribute.
+    </li>
+  </ul>
+
+
+</ngbd-overview-section>
+
+<ngbd-overview-section [section]="sections['customization']">
+  <p>
```

Comment:
I would myself remove this section introduction, and directly create 2 `<ngbd-overview-section>` foreach one.
It will make the doc more concise, and we will have navigation links in the side topic list for free.